components/dev_to/sources/fresh-stories-by-tag/fresh-stories-by-tag.mjs

```diff
@@ -29,14 +21,14 @@ export default {
   },
   dedupe: "greatest",
   async run() {
-    const url = `https://dev.to/api/articles?per_page=1000&top=1&tag=${encodeURIComponent(this.tag)}`;
-    const data = (await axios({
-      method: "get",
-      url,
-      headers: {
-        "api-key": `${this.devTo.$auth.api_key}`,
+    const { data } = await this.devTo.callApi({
```

Comment:
Could you please create a method on `app` instead of calling `callApi`? Just for better readability. 
For example, you can create a method called `getArticles` at `app` with this code, and just call `this.devTo.getArticles` from here